_extras/design.md

```diff
@@ -118,10 +118,84 @@ How do I...
 ### Variables and Assignment (9:15)
 
 *   Teaching: 10 min
+
+Basic assignment of a value.  What kind of thing is it?  Can assign results
+of computation and use a variable on the right to assign to the left.
+
+```
+a = 3
+print(a)
+type(a)
+a = a + 60
+print(a)
+```
```

Comment:
We prefer to close with `~~~`, and then the next line is `{: .python}` to style it (because that's how Kramdown prefers styles on headings, tables, and other things).